### PR TITLE
docs: correct some 404 links

### PR DESCRIPTION
**Pull request by ocavue**
## Checklist

- [x] I have read the [**contributing**](https://github.com/remirror/remirror/blob/master/docs/contributing.md) document.
- [x] My code follows the code style of this project and `pnpm fix` completed successfully.
- [x] I have updated the documentation where necessary.
- [x] New code is unit tested and all current tests pass when running `pnpm test` .

### CODE REVIEW

File: .changeset/young-plants-help.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 '@remirror/extension-tables': major
 ---
 
-This package is now deprecated. It has been replaced with [`@remirror/preset-table`](https://github.com/remirror/remirror/tree/next/@remirror/preset-tables).
+This package is now deprecated. It has been replaced with [`@remirror/preset-table`](https://www.npmjs.com/package/@remirror/extension-tables).
```
Comment (ifiokjr) [inline .changeset/young-plants-help.md:5]:
```suggestion
This package is now deprecated. It has been replaced with [`@remirror/preset-table`](https://www.npmjs.com/package/@remirror/preset-table).
```

File: support/deprecated/extension-tables/CHANGELOG.md [hunk 2]
```diff
@@ -29,7 +29,7 @@
 - 7b817ac2: Rename all types and interfaces postfixed with `Params` to use the postfix `Parameter`.
   If your code was importing any matching interface you will need to update the name.
 - 304db6ba: This package is now deprecated. It has been replaced with
-  [`@remirror/preset-table`](https://github.com/remirror/remirror/tree/next/@remirror/preset-tables).
+  [`@remirror/preset-table`](https://www.npmjs.com/package/@remirror/extension-tables).
```
Comment (ifiokjr) [inline support/deprecated/extension-tables/CHANGELOG.md:32]:
```suggestion
  [`@remirror/preset-table`](https://www.npmjs.com/package/@remirror/preset-table).
```

File: support/deprecated/extension-tables/readme.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # @remirror/extension-tables
 
 This package is deprecated. It has been replaced with
-[`@remirror/preset-table`](https://github.com/remirror/remirror/tree/next/@remirror/preset-tables).
+[`@remirror/preset-table`](https://www.npmjs.com/package/@remirror/extension-tables).
```
Comment (ifiokjr) [inline support/deprecated/extension-tables/readme.md:4]:
```suggestion
[`@remirror/preset-table`](https://www.npmjs.com/package/@remirror/preset-table).
```

**Comment by ocavue** [comment]
Change all github links from `master` and `next` to `HEAD` so that we don't need to edit these links again when we switch back the default branch from `next` to `master`.

```diff
- https://github.com/remirror/remirror/tree/master/xxx
+ https://github.com/remirror/remirror/tree/HEAD/xxx
```

```diff
- https://github.com/remirror/remirror/tree/next/xxx
+ https://github.com/remirror/remirror/tree/HEAD/xxx
```

**Comment by ocavue** [comment]
> Perhaps provide a changeset since it touches the npm packages repository field.

done

**Pull request merged**